front/app/containers/Admin/inspirationHub/Filters/Dates.tsx

```diff
@@ -5,14 +5,15 @@ import { format } from 'date-fns';
 import DateRangePicker from 'components/admin/DatePickers/DateRangePicker';
 
 import { trackEventByName } from 'utils/analytics';
+import { parseBackendDateString } from 'utils/dateUtils';
 
 import { useRansackParam, setRansackParam } from '../utils';
 
 import tracks from './tracks';
 
 const toDate = (str?: string) => {
   if (!str) return;
-  return new Date(str);
+  return parseBackendDateString(str);
```

Comment:
Found this bug today, when working on the similar date filter for the new projects page. So fixed it here as well